src/shared/services/json-util.service.ts

```diff
@@ -27,15 +27,25 @@ export class JsonUtilService {
 
   /**
    * Returns value of the property located in dot separated path of json.
+   * Finds the appropriate key if items and properties are not in the path.
    */
   getValueInPath(json: any, path: string): any {
```

Comment:
this actually is almost identical with https://github.com/inveniosoftware-contrib/ng2-json-editor/blob/master/src/shared/services/json-schema.service.ts#L39, can you have a look as well? If not we can maybe change it a little bit because `json-schema.service` makes more sense for this kind of utility function.

Since this utility function is already used for normal json we can leave it as it is.